server/boot/passportConfig.js

```diff
@@ -0,0 +1,37 @@
+const passport = require("passport");
+const GoogleStrategy = require("passport-google-oauth20").Strategy;
+const User = require("../models/User");
+
+//configure passport.js
+module.exports = function () {
+  passport.use(
+    new GoogleStrategy(
+      {
+        clientID: process.env.GOOGLE_CLIENT_ID,
+        clientSecret: process.env.GOOGLE_CLIENT_SECRET,
+        callbackURL: "/auth/google/callback",
+        passReqToCallback: true,
+      },
+      async function (req, accessToken, refreshToken, profile, cb) {
+        //add accessToken to session
+        req.session.accessToken = accessToken;
+
+        let user = await User.findOne({ googleId: profile.id });
+        if (!user) {
+          user = await User.create({
+            googleId: profile.id,
+            gmail: profile.emails[0].value,
```

Comment:
This probably could just be called `email` instead of `gmail`

Reply:
fix gmail and user model eb6556e9d2872e75897e1501c59377847cc3ebe9